01_data_exploration.ipynb

```diff
@@ -10,17 +10,17 @@
     "# 1. Data Exploration\n",
```

Comment:
<p>Could: Have some small contextual elaboration to example these issues e.g. a classifier that predicts "no rotors" regardless of input will have an accuracy of 97.43% </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/informatics-lab/ml_weather_tutorial/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I've added some additional explanatory text.</p>